File: front-end/src/model/order.ts
export class Order {
    id:number;
    name:string;
    data_created:Date;
    data_upated:Date;
    price:number;
    description:string;


    constructor(id: number, name: string, data_created: Date, data_upated: Date, price: number, description: string) {
        this.id = id;
        this.name = name;
        this.data_created = data_created;
        this.data_upated = data_upated;
        this.price = price;
        this.description = description;
    }
}

File: backend/spring-resturant/src/main/java/com/spring/resturant/model/Order.java
package com.spring.resturant.model;

import com.fasterxml.jackson.annotation.JsonIgnore;

import jakarta.persistence.Entity;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.Lob;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name="orders")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Order extends CategoryOrder{
	
	@Lob
	private String description;
	private int price;
	
	@JsonIgnore
	@ManyToOne()
	@JoinColumn(name="id_category")
	private Category category;
	
	

}
